pcsa_GHN/models.py

```diff
@@ -12,14 +10,22 @@
 
 class Contact(models.Model):
     office_name = models.CharField(max_length=200)
-    contact_number = models.BigIntegerField()
+    contact_number = models.CharField(
+        max_length=20,
+        validators=[
+            RegexValidator(
```

Comment:
@Tahirih-Natang why can't we use a simpler regex as shared in the Stackoverflow answer here https://stackoverflow.com/questions/6478875/regular-expression-matching-e-164-formatted-phone-numbers

Reply:
@sara-02 all changes have been made as requested